Vertical graphs are now working

diogoscf opened the pull request:
Fixes #167 
Hard to find the cause, but easy to fix!
The vertical graphs are somewhat ugly, but they already were before they got broken.

ftyers commented:
@diogoscf can you add a screenshot ?

diogoscf commented:
How it looks now:
![new_vertical](https://user-images.githubusercontent.com/26374538/34914301-49e59ba6-f910-11e7-82f9-14aba57d9dfa.PNG)
For comparison, how it looked in the last commit before it broke:
![old_vertical](https://user-images.githubusercontent.com/26374538/34914302-5b764046-f910-11e7-8401-e407753d4fd8.PNG)

gstark0 commented on standalone/lib/visualiser.js:
```diff
@@ -697,7 +697,7 @@ function cleanEdges() {
 function setEdgePosition(thisEdge, thisHeight, coef, diff) {
 	if (!LEFT_TO_RIGHT) {coef *= -1}; // support for RTL
-	if (VERT_ALIGNMENT) {edgeDep.ctrl = [45, 45, 45, 45]};
+	//if (VERT_ALIGNMENT) {edgeDep.ctrl = [45, 45, 45, 45]};
```
It's the same as before, why is it commented?

jonorthwash commented:
Could you show a screenshot of connected nodes too?

jonorthwash commented:
Also, which commit broke it?

diogoscf commented:
The commit that broke it was 596379f

diogoscf commented:
It's a bit broken:

![new_vertical_withnodes](https://user-images.githubusercontent.com/26374538/34919945-120f79f8-f96b-11e7-9083-4ac8199be61e.PNG)

But it wasn't before... Hadn't noticed that...

![old-vertical_withnodes](https://user-images.githubusercontent.com/26374538/34919956-388843a8-f96b-11e7-9694-ee51d935def6.PNG)

jonorthwash commented:
Oh, I know why it's doing that.  It's staggering the arcs so that incoming and outgoing arcs don't overlap, which is by design.  Could you test it out with some real trees from a treebank (and share screenshots)?

maryszmary commented:
It doesn't work for (e.g.) this tree:
> 1	N'	ne	ADV	adv	Polarity=Neg	4	advmod	_	_
2	int	bezañ	VERB	vblex	Mood=Ind|Number=Plur|Person=3|Tense=Pres|VerbForm=Fin	4	aux	_	_
3	ket	ket	ADV	adv	_	4	advmod	_	_
4	aet	mont	VERB	vblex	Tense=Past|VerbForm=Part	0	root	_	_
5	war-raok	war-raok	ADV	adv	_	4	advmod	_	_
6	.	.	PUNCT	sent	_	4	punct	_	_

giving the following error messages from cytoscape:

> The style property `text-margin-x: ` is invalid

> cytoscape.min.js:1 Do not assign mappings to elements without corresponding data (e.g. ele `ep01` for property `text-margin-x` with data field `length`); try a `[length]` selector to limit scope to elements with `length` defined

> cytoscape.min.js:10 Uncaught TypeError: Cannot set property 'mapping' of null

(horizontal alignment does work):
![image](https://user-images.githubusercontent.com/16348652/34941796-4e608032-fa06-11e7-8150-4d8e428121f8.png)

diogoscf commented:
Huh. Those are the errors it gave before I fixed it and thay stopped after I fixed the code. Are you sure you aren’t using the code from before the fix @maryszmary ?

diogoscf commented:
And a two more examples:
![somenode](https://user-images.githubusercontent.com/26374538/34958172-fe1019b4-fa30-11e7-98a3-1f53ca02ccf6.PNG)
![othernode](https://user-images.githubusercontent.com/26374538/34958174-00bbe56c-fa31-11e7-86bb-cf47cc4d9bd1.PNG)

diogoscf commented:
They now look good again!
Examples:
![othernodes](https://user-images.githubusercontent.com/26374538/35011313-f00f7488-fb05-11e7-9353-415823ca2fbc.PNG)
![somenodes](https://user-images.githubusercontent.com/26374538/35011317-f1c08876-fb05-11e7-84d8-3c7f850797ed.PNG)
![extranodes](https://user-images.githubusercontent.com/26374538/35011322-f38d6ba6-fb05-11e7-93f6-d02fc36418f9.PNG)

jonorthwash commented:
The arrows with the relation tags are pretty meaningless like this...

maryszmary commented:
@diogoscf , my mistake: it works. Thank you!
